### PR TITLE
Add HTS HIV data

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: data/HIV/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  type: SMILES
+  description: SMILES
+license: CC BY 4.0
+links:
+- url: https://wiki.nci.nih.gov/display/NCIDTPdata/AIDS+Antiviral+Screen+Data
+  description: data source
+- url: https://doi.org/10.1039/C7SC02664A
+  description: corresponding publication
+num_points: 1484
+url: https://tdcommons.ai/single_pred_tasks/hts/#hiv
```
Comment (kjappelbaum) [inline data/HIV/meta.yaml:27]:
```suggestion
num_points: 1484
```

Comment (kjappelbaum) [reply data/HIV/meta.yaml:27]: Perhaps add it under `links`?

**Review by phalem — COMMENTED** [review]
I remove url and put it into links

**Review by phalem — COMMENTED** [review]
Add benchmark field

**Comment by MicPie** [comment]
URL 403

**Pull request merged**